### PR TITLE
Add AddLogicalUserList code example

### PR DESCRIPTION
**Pull request by PierrickVoulet**
Change-Id: Ifd416b1bff7157658397eb05295d7fb92532b2d9

### CODE REVIEW

File: examples/Remarketing/AddUnionUserList.php [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+use Google\Ads\GoogleAds\V4\Resources\UserList;
+use Google\Ads\GoogleAds\V4\Services\UserListOperation;
+use Google\ApiCore\ApiException;
+use Google\Protobuf\StringValue;
+
+/**
+ * Creates a combination user list containing users that are present on any one of the provided user
+ * lists.
+ */
+class AddUnionUserList
```
Comment (fiboknacky) [inline examples/Remarketing/AddUnionUserList.php:45]: I wonder why we name this "UnionUserList".
Nowhere in the code that has the word "union". 
Instead, we call this kind of user list "combination user list" in many places.

Comment (PierrickVoulet) [reply examples/Remarketing/AddUnionUserList.php:45]: @devchas any background you could provide on this as you wrote the golden?

Comment (devchas) [reply examples/Remarketing/AddUnionUserList.php:45]: Please see the [conversation history](https://github.com/googleads/google-ads-java/pull/241) on the PR - specifically comments made by Nick on Mar 3.

Comment (PierrickVoulet) [reply examples/Remarketing/AddUnionUserList.php:45]: Thanks @devchas for the link. SGTM, any strong feelings about this @fiboknacky ?

Comment (PierrickVoulet) [reply examples/Remarketing/AddUnionUserList.php:45]: @fiboknacky Is it OK with you if I keep it like that?

Comment (fiboknacky) [reply examples/Remarketing/AddUnionUserList.php:45]: If Devin doesn't object to this, I prefer we use the name CombinationUserList instead.
We use that word in so many places throughout the code, so why do we just name the example differently?

Comment (devchas) [reply examples/Remarketing/AddUnionUserList.php:45]: I am okay with changing the name of the example. However, I think AddCombinationUserList is too similar to AddCombinedRuleUserList and may cause confusion. Part of the reason we ended up with the name AddUnionUserList is that I wrote several UserList-related examples all at the same time for a guide, and we wanted to make sure that all of the names were different enough as not to cause confusion.

Comment (PierrickVoulet) [reply examples/Remarketing/AddUnionUserList.php:45]: What about "AddLogicalUserList" instead?

Comment (devchas) [reply examples/Remarketing/AddUnionUserList.php:45]: Okay with me.

Comment (PierrickVoulet) [reply examples/Remarketing/AddUnionUserList.php:45]: @fiboknacky If OK with you, I will change to AddLogicalUserList as it seems to be the best tradeoff.

**Comment by PierrickVoulet** [comment]
> Have you changed the example name? I've not seen it's changed yet.

Done, it should be good this time.

**Pull request merged**